dist/index.html

```diff
@@ -1,19 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="https://fonts.googleapis.com/css2?family=Lora&display=swap" rel="stylesheet">
-    <link rel="stylesheet"
-        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lora&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="stylesheet" href="/src/style.css">
 
     <title>Movies Database</title>
-<script defer src="main.js"></script></head>
-
-<body>
-    <h1>Group Capstone</h1>
-
+  <script defer src="main.js"></script></head>
+  <body>
+    <header id="navigation">
+      <a href="#" class="logo">LOGO</a>
+      <a href="#" class="logo">Latest Movies/Shows</a>
+      <nav id="other-interfaces">
+        <a href="#" class="interface">Tv Shows</a>
+        <a href="#" class="interface">Movies</a>
+      </nav>
+    </header>
```

Comment:
Header and necessary elements are successfully created 🎯 
Good job 👏
It's time to merge it and start the next task. Happy coding 👩🏻‍💻